compiler/codegen/expr.cpp

```diff
@@ -4743,6 +4745,7 @@ GenRet CallExpr::codegenPrimitive() {
       // should deref the 'src'.
       if (src->symbol->hasFlag(FLAG_REF) &&
           dst->symbol->hasFlag(FLAG_REF) == false) {
+        // (*src)
```

Comment:
I'm unsure of the value of this and the above comment changes. Usually when I add such comments they reflect my increasing understanding, but I'm not sure how helpful they'd be to others. I recommend you either remove them or turn them into more words.